### ISSUE TITLE
Missing Modules are output when multiprocessing is import

### ISSUE THREAD

**Issue by NT-Parabox** [py2exe/py2exe#60]
Hello.
I am creating a Python program and converting it to an exe file in the following environment.

- Windows10 64bit
- Python 3.7.9
- py2exe 0.10.1.0 (py2exe-0.10.1.0-cp37-none-win32.whl)

```cmd
C:\>pip3 list
Package     Version
----------- -------
ezdxf       0.14.2
geomdl      5.3.0
ordereddict 1.1
pip         20.1.1
py2exe      0.10.1.0
pyparsing   2.4.7
pyreadline  2.1
pywin32     300
setuptools  47.1.0
wheel       0.36.2
```

The created python program itself is working fine,
I found that the following warning occurs when converting using py2exe.

```python
#!python3.7
# coding: utf-8

from distutils.core import setup
import py2exe

option = {
    'compressed': 1,
    'optimize': 2,
    'bundle_files': 3,
}

setup(
    options = {
        'py2exe': option,
    },
    console = [
        {'script': 'program.py'}
    ],
    zipfile = None,
)
```

```python
  1 missing Modules
  ------------------
? multiprocessing.Pool                imported from multiprocessing.context
```

The following is an example of a simple program that causes a warning when converting with py2exe.

```python
from multiprocessing import Pool

def sub(x):
    return x * x


def main():
    p = Pool(4)
    result = p.map(sub, range(10)) 
    print(result)


if __name__ == "__main__":
    main()
```

When using multiprocessing, the output is that the module cannot be found.
Is it a phenomenon caused by the following problems?

> The modulefinder does not fully support PEP420 implicit namespace packages.

I would like to use the exe file after removing this warning if possible.
If it's difficult to get rid of the warning, is it okay to ignore this warning?

**Comment by albertosottile** [comment]
Hello, thanks for reporting this. It took me a while to figure out what was happening. 

So, I think your question is about this entry in the build log:
```
running py2exe

  4 missing Modules
  ------------------
[...]
? multiprocessing.Pool                imported from __SCRIPT__
```

Usually, these lines are safe to ignore provided that your packaged code runs well. In this case, `multiprocessing` mimics a lot of its structure at the package level, probably to avoid having to type too much while using it. The "correct" way to import `Pool` should be `from multiprocessing.pool import Pool`. You can verify that the warning disappears if you change your script accordingly.

That being said, it took me a while to find this issue because your test script **cannot run** when packed via `py2exe`. At first I thought this was, as usual, a bug on my side, so I spent some time trying to make it work. Lately, I found that `multiprocessing` actually provides a dedicated function that must be called from inside any frozen executable (not just one built with py2exe, but also with any other tool): [multiprocessing.freeze_support()](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.freeze_support).

By editing your code according to the documentation, I was indeed able to run the test and verify that the code was functional. 

In summary, you can either safely ignore the "missing Modules" warning or slightly adjust your import statement to make it disappear. However, you must add a call to `multiprocessing.freeze_support()` in your code to make it work when packed with `py2exe`.
